ci/v2/test/core.flaky.yml

```diff
@@ -0,0 +1,12 @@
+targets: 
```

Comment:
while we are waiting for yq to be installed, can we change this to `flaky_tests:`, and maybe rename the file name to `core.tests.yml` or something? so that this is going towards general per-team test config data, rather than just for flaky test tracking?